dnsapi/dns_kappernet.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env sh
+
+# kapper.net domain api
+# for further questions please contact: support@kapper.net
+# please report issues here: https://github.com/acmesh-official/acme.sh/issues/2977
+
+#KAPPERNETDNS_Key="yourKAPPERNETapikey"
+#KAPPERNETDNS_Secret="yourKAPPERNETapisecret" 
+
+KAPPERNETDNS_Api="https://dnspanel.kapper.net/API/1.2?APIKey=$KAPPERNETDNS_Key&APISecret=$KAPPERNETDNS_Secret"
+
+###############################################################################
+# called with 
+# fullhostname: something.example.com
+# txtvalue:     someacmegenerated string
+dns_kappernet_add() {
+  fullhostname=$1
+  txtvalue=$2
+
+  if [ -z "$KAPPERNETDNS_Key" ] || [ -z "$KAPPERNETDNS_Secret" ]; then
+    KAPPERNETDNS_Key=""
+    KAPPERNETDNS_Secret=""
+    _err "You haven't defined kapper.net api key and secret yet."
+    _err "Please send us mail to support@kapper.net get your key and secret."
+    return 1
+  fi
+
+  #store the api key and email to the account conf file.
+  _saveaccountconf KAPPERNETDNS_Key "$KAPPERNETDNS_Key"
```

Comment:
did you read this section?
https://github.com/acmesh-official/acme.sh/wiki/DNS-API-Dev-Guide#2-you-must-save-your-username-and-password-in-the-add-function

Reply:
well, as you are quite non-specific - yes - the section was read since several months (and before years) multiple times.

I can only guess you want an "_saveaccountconf_mutable" instead of the previous (and therefore by used by us) "_saveaccountconf" - but I can only guess as you are very non-specific in your feedback.